### PR TITLE
Document new features in version 1.1.1

### PR DESCRIPTION
**Pull request by settermjd**
## Description
This change documents the changes to the app as outlined in:

- https://github.com/owncloud/ios-app/pull/478
- https://github.com/owncloud/ios-app/pull/474
- https://github.com/owncloud/ios-app/pull/469

These changes were requested in:

- https://github.com/owncloud/docs/issues/1900, and
- https://github.com/owncloud/docs/issues/1928

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by settermjd** [comment]
This PR goes in-tandem with #527.

File: docs/modules/ROOT/pages/ios_files.adoc [hunk 1]
```diff
@@ -52,7 +55,32 @@ To go back to the parent folder, tap btn:[Back] in the top left-hand corner.
...
 
-image:21_File_list_parent.png[Navigating folders in ownCloud's iOS app.]
+image:21_File_list_parent.png[Navigating folders in ownCloud's iOS app]
+
+== Navigate Files
+
+When navigating files, as you would expect, you can scroll up and down the files and folders list.
+In addition, you can also use the index bar, highlighted in the screenshot below, to speed up traversing through files and folders.
+As you slide your finger over each letter, you’ll jump to the first file (or folder) that begins with that letter.
+
```
Comment (hosy) [inline docs/modules/ROOT/pages/ios_files.adoc]: Please note: The index bar is only visible, if the sort order is `Name`.

File: docs/modules/ROOT/pages/ios_settings.adoc [hunk 2]
```diff
@@ -71,9 +71,9 @@ image:84_Settings_themes.png[84_Settings_themes.png]
 |Dark theme
 |Light theme
 
-a|image:themes-ownCloud-Classic.png[ownCloud iOS App - Classic theme]
-a|image:themes-ownCloud-Dark.png[ownCloud iOS App - Dark theme]
-a|image:themes-ownCloud-Light.png[ownCloud iOS App - Light theme]
+a|image:themes/classic.png[ownCloud iOS App - Classic theme]
+a|image:themes/dark.png[ownCloud iOS App - Dark theme]
+a|image:themes/light.png[ownCloud iOS App - Light theme]
 |===
```
Comment (hosy) [inline docs/modules/ROOT/pages/ios_settings.adoc]: Would be good, to describe what the happens, when theme `System Appearance` is enabled and what this means.

**Comment by settermjd** [comment]
Thanks for the feedback, @hosy.

File: docs/modules/ROOT/pages/ios_settings.adoc (hunk 2 above)
Comment (settermjd) [reply docs/modules/ROOT/pages/ios_settings.adoc]: Can you provide more information about that, please?

Comment (hosy) [reply docs/modules/ROOT/pages/ios_settings.adoc]: When `System Appearance` is selected, ownCloud will automatically use the selected iOS system theme (light or dark) to reflect the system UI. Setting `System Appearance` is only available up from iOS 13.

**Comment by settermjd** [comment]
@hosy, can you help with fixing the breaking build, pls?

**Pull request merged**